package.json

```diff
@@ -9,6 +9,7 @@
     "lint": "eslint --ext .js --ext .jsx .",
     "lint_fix": "eslint --fix --ext .js --ext .jsx .",
     "prepare": "npm run build",
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post",
```

Comment:
Since `v9.0.0` this has just been `semantic-release`

![image](https://user-images.githubusercontent.com/8136030/39013641-fb56f612-43e5-11e8-86f8-8f1619af6d3b.png)

Reply:
Gracias, I had just copied this from your initial PR to start